### PR TITLE
chore: Bump node/yarn version & removing obsolete script

### PR DESCRIPTION
**Pull request by kushthedude**
- We had to revert back to node 8 due to some failing tests which is not the case right now
- We have netlify as our deployer and we no longer need surge script for deployment

### CODE REVIEW

**Comment by kushthedude** [comment]
Not to be merged until next release. 
Need thorough testing before making into production.

**Comment by iamareebjamal** [comment]
Update docker image

File: package.json [hunk 1]
```diff
@@ -131,8 +131,8 @@
     "xgettext-template": "^3.4.2"
   },
   "engines": {
-    "node": "8.* || >= 10.*",
-    "yarn": ">= 1.0.0"
+    "node": ">= 10.*",
```
Comment (iamareebjamal) [inline package.json]: Why not 12

Comment (kushthedude) [reply package.json]: @iamareebjamal I am unable to make netlify utilise the latest version as it is using the cached version of node which is 10 used in recent PR builds

Comment (kushthedude) [reply package.json]: @iamareebjamal Ping!

File: Dockerfile [hunk 2]
```diff
@@ -20,7 +20,7 @@ RUN node scripts/l10n.js generate && \
 ##
 ##
 
-FROM node:10-alpine
+FROM node:12.14.1-alpine3.9
```
Comment (iamareebjamal) [inline Dockerfile]: 12-alpine doesn't work?

Comment (kushthedude) [reply Dockerfile]: It does work.

**Comment by iamareebjamal** [comment]
netlify deploy is failing

**Comment by kushthedude** [comment]
Itb6is because netlify is using cached version of last deployment which is
node10 . Hence it is failing.
Any solution for this?

**Comment by kushthedude** [comment]
@iamareebjamal Please review & merge

**Pull request merged**